### ISSUE TITLE
Update veeam-backup-and-replication to version 12

### ISSUE THREAD

**Issue by tliknes** [mkevenaar/chocolatey-packages#173]
Veeam Backup and Replication V12 is released
https://go.veeam.com/v12

**Comment by mkevenaar** [comment]
Yeah, I know. I need to rework the packages because everything is changed.

Hope that I can do this over the weekend

**Comment by tliknes** [comment]
No problem!
Thanks!

**Comment by neiljackson1984** [comment]
It looks like Veeam has put the page containing their download URLs (https://www.veeam.com/download-version.html) behind a password prompt.  This is probably preventing the update script (https://github.com/mkevenaar/chocolatey-packages/blob/master/automatic/veeam-backup-and-replication-iso/update.ps1) from scraping the page to get the latest version number and corresponding URL of the .iso file.

**Comment by opoplawski** [comment]
Seems like this can be closed.
